README.org

```diff
@@ -1,3 +1,5 @@
+
+
```

Comment:
Please remove this.

Reply:
Either I am missing something or GitHub isn't displaying this properly, because that review comment seems to be referring to two blank lines, which I don't see in the actual file.